.github/workflows/cxx-build.yml

```diff
@@ -121,27 +119,58 @@ jobs:
           key: deps-node-modules-${{ hashFiles('package-lock.json') }}
 
       - name: Install npm dependencies
-        run: npm ci
+        run: |
+          apt update
+          apt install -y cmake
+          npm install
 
       - name: Generate core SDK source code
         run: |
           .github/workflows/utils.sh generate_cpp_core_sdk_source_code
 
-      - name: Build CXX Core SDK 
+      - name: Build CXX Core SDK
         run: |
+          apt install -y python3-pip python3.11-venv
+          python3 -m venv firebolt-env
+          source firebolt-env/bin/activate
+          pip install gcovr
+
           .github/workflows/utils.sh build_core_cpp_sdk
 
+      - name: Test and Generate coverage report
+        run: .github/workflows/utils.sh generate_core_sdk_coverage_report
+
+      - name: Display coverage report
+        uses: irongut/CodeCoverageSummary@v1.3.0
+        with:
+          filename: coverage/coverage.cobertura.xml
+          badge: true
+          fail_below_min: true
```

Comment:
For now let's not fail. I want to eventually but not right now.

Reply:
even if ctest fail. Git action steps won't be interrupted for now.